lessons/commutative-property/langs/en-us/index.adoc

```diff
@@ -0,0 +1,116 @@
+= The Commutative Property
+
+@description{This lesson introduces students to the Commutative Property by way of Circle Transformations.}
```

Comment:
Instead of "by way of Circle Transformations" how about "using Circles of Evaluation"